<launch>
    <param name="apriltag_docking/lookahead_dist_multiplier" type="double" value="0.6" />
    <param name="apriltag_docking/rotational_vel_gain" type="double" value="1.5" />
    <param name="apriltag_docking/linear_vel_gain" type="double" value="0.1" />
    <param name="apriltag_docking/goal_tolerance" type="double" value="0.1" /> <!-- 0.33-->
    <param name="apriltag_docking/tag_on_ceiling" type="bool" value="True" />
    <node name="apriltag_docking" pkg="apriltag_docking_ros" type="apriltag_docking_action_server.py" output="screen"/>
</launch>